src/components/AnchorList/style.js

```diff
@@ -0,0 +1,111 @@
+import styled from '@emotion/styled'
+import Title from '../Utilities/Title'
+
+export const AncherContainer = styled.div`
+  margin-top: 102px;
+`
+export const AnchorTitle = styled(Title)`
+  padding-bottom: 16px;
+`
+export const AnchorContent = styled.div`
+  border-top: 6px solid ${(props) => props.theme.colors.black};
+  border-bottom: 6px solid ${(props) => props.theme.colors.black};
+  padding-top: 13px;
+  padding-bottom: 30px;
+
+  @media (min-width: ${(props) => props.theme.breakpoints.md}) {
+    border-top: 12px solid ${(props) => props.theme.colors.black};
+    border-bottom: 12px solid ${(props) => props.theme.colors.black};
+    padding-bottom: 109px;
+  }
+`
+
+export const AnchorLinksMob = styled.div`
+  display: flex;
+  flex-direction: row;
+  padding-top: 9px;
+
+  .anchor_links {
+    display: flex;
+    color: ${(props) => props.theme.colors.black};
+    font-family: ${(props) => props.theme.fontFamily.primary};
+    font-size: ${(props) => props.theme.fonts.body};
+    text-decoration: none;
+  }
+`
+
+export const AnchorLinks = styled.div`
+  display: flex;
+  flex-direction: row;
+  padding-bottom: 9px;
+  align-items: center;
+
+  .anchor_links {
+    display: flex;
+    color: ${(props) => props.theme.colors.black};
+    font-family: ${(props) => props.theme.fontFamily.primary};
+    font-size: ${(props) => props.theme.fonts.body};
```

Comment:
Is this the only place on the site that this font-size is being used? maybe it would be worth it creating a component type for this?

Reply:
Yes this is the only Place, but I can create a component type if it is better ?